lib/actions/jlinkTargetActions.js

```diff
@@ -319,47 +353,78 @@ export const loadDeviceInfo = (
     // Update target info according to detected device info
     dispatch(modemKnownAction(info.family === 91));
 
+    // By default readback protection is none
+    let readbackProtection = ReadbackProtectionType[nrfjprog.NONE];
+
     let deviceInfo = getDeviceInfoByJprog(info);
+    readbackProtection = await new Promise((resolve, reject) =>
```

Comment:
If you expect a rejection this must be wrapped in a try catch. Same a few lines later.